res/lang/rooms/rooms_en.txt

```diff
@@ -148,7 +148,7 @@ INGAME_HELP_TITLE_SCREEN_ARCHIVE = Archive
 INGAME_HELP_TEXT_SCREEN_ROOMAGENCY  = If one studio is not sufficient for your productions, you can rent further rooms here or cancel existing contracts.\nIf no free rooms are available, you'll have to do somwthing about it...
 INGAME_HELP_TITLE_SCREEN_ROOMAGENCY = Room Agency
 
-INGAME_HELP_TEXT_SCREEN_ROOMBOARD  = Visitors rely on the information from the room board before entering the elevator. Deliberate disorder is corrected once a day.
+INGAME_HELP_TEXT_SCREEN_ROOMBOARD  = Visitors rely on the information from the room board before entering the elevator. Intentional disarray is corrected once a day.
```

Comment:
feel free to improve this hint (also in German).
Maybe: "Intentional disarray which might cause visitors to end up in the wrong room..."